src/Common/src/Common/Util/MimeType.cs

```diff
@@ -144,7 +144,7 @@ protected void CheckParameters(string attribute, string value)
             if (PARAM_CHARSET.Equals(attribute))
             {
                 value = Unquote(value);
-                Encoding.GetEncoding(value);
+                _ = Encoding.GetEncoding(value);
```

Comment:
Probably ignore this suggestion - I dont see value ...

Reply:
This suppresses the R# warning that the return value of a pure function is unused, which is usually an indication of a mistake. In this case however, I believe the try/catch block is used to detect failure, whereas the actual returned encoding is irrelevant here.

Adding the `_` (discard) expresses that ignoring the return value is intentional. By being explicit about that, I believe this improves code readability: Future readers won't have to wonder whether this was intentional.

Based on the above, are you okay with keeping this?

Reply:
Ok sounds good.